src/ranged.cpp

```diff
@@ -648,6 +648,9 @@ static double confidence_estimate( int range, double target_size, dispersion_sou
     // is not doing Gaussian integration in their head while aiming.  The result gives the player
     // correct relative measures of chance to hit, and corresponds with the actual distribution at
     // min, max, and mean.
+	if (range == 0) {
```

Comment:
New code have to comply to current formatting rules from [CODE_STYLE.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CODE_STYLE.md).
Namely, the proper formatting is as follows:
```if( range == 0 ) {```

Reply:
Ouch, VS auto format caught me there. Corrected now